### ISSUE TITLE
Consider moving away from verbs in the sidebar

### ISSUE THREAD

**Issue by fasterthanlime** [itchio/itch#188]
I feel like that's partly what #184 is about re: "the app is hard to parse/navigate" ?

I like using verbs because it was precise. For example, `Developed` used to be called `Dashboard` and users were very confused by it because they thought it was a dashboard for everyone, not just for developers. Whereas `Developed`, while awkward, gets the point across.

As far as `Owned` goes, it could be `Bought`, but it also includes claimed games, so that's not really accurate. It could be named `Library` (like every other platform) or `My purchases` (like the website), but I'm not happy with either of those. `Library` sounds like it could "the entire itch.io library" or someone else's library that I'm visiting. It's not really a purchase list either, ie. you can't see date of purchase, amount, etc.

So, I'm not convinced we should move aways from verbs in the sidebar, but I'll consider it, and at any rate, the onboarding experience should be better — there could be a header on the top of the game grid when you first open a tab, with a quick explanation of what goes on. In fact we already have that as a placeholder, see https://github.com/itchio/itch/issues/119#issuecomment-165315571 — but it only appears if the game grid is empty, so I'm pretty sure Jeff missed most of that (and so did many others).

**Comment by jvdgoot** [comment]
![2015-12-18 22_57_04-itch](https://cloud.githubusercontent.com/assets/571375/11908227/c47da5c4-a5da-11e5-96fa-08c597c847ec.png)

Okay, the main problem here is that the context for section 1 & 2 are unclear and are hierarchically on the same level visually, when that's not really the case.

Sure, the three headers "Developed", "Owned" and "Installed" are three types of games owned by jvdgoot, but "installed" can be a subsection of Owned or Developed, technically but they are presented on the same level in the interface.

Then the number 2 section is are collections I presumably created or faved, which are seperated from the jvdgoot, but supposedly still connected to it.

More sense to me would make a Games overall category, which have filters applied to them such as "Developed", "Owned", "Installed" or part of some collection.

It is a semantic difference, but one that could be more clear and clarified in the user interface.

**Comment by fasterthanlime** [comment]
So if I got this right:
- A 'Games' header is necessary somewhere (but than what of non-game content? people can sell books, soundtracks, tools on itch.io and soon the app will support them too)
- The separator between 'installed' and collections is unnecessary and confusing

Btw, the star in those last two items is for "featured collections", not collections you've fav'd. It was a last minute addition to temporarily help with #100 — so I just took the first icon that made a little sense to me. Can you think of an icon that would make more sense? Ideally, it'd be opt-out, too (if you don't want to see our collections you don't have to, it's just to show how it all works).

Fav'ing/following/subscribing to collections is a feature I've been wanting for a while, as soon as there's API support for that, the app will definitely support it.

**Comment by fasterthanlime** [comment]
The strings have been changed, we're moving towards nouns, cf. https://weblate.itch.ovh/translate/itch/itch/en/?q=sidebar&search=substring&context=on&type=all&ignored=False
